### PR TITLE
Add Doctrine Migrations and migrations configuration factory / fixes #28

### LINKED ISSUE #28: Idea - Migrations factory
Got asked after my talk about whether we support Doctrine Migrations, which I realise we implemented manually on other projects with a factory.

This is a placeholder to remind me to add that factory (optionally, of course) into here so others can benefit :+1:

### PR DESCRIPTION
**Pull request by luckals**
I think it would useful to integrate doctrine migrations.

### CODE REVIEW

**Comment by DASPRiD** [comment]
You should add a usage example for that in the README. Also, migrations should end up in the suggest section, not everyone wants to use it.

**Comment by luckals** [comment]
Thanks for the feedback! I have now added an usage example in the README file und moved migrations from require into suggest in the composer.json

File: composer.json [hunk 1]
```diff
@@ -1,28 +1,31 @@
 {
-    "name": "dasprid/container-interop-doctrine",
```
Comment (DASPRiD) [inline composer.json:2]: Why was the formatting of this file changes? This is auto-generated by composer and thus should not have that many changes.

Comment (luckals) [reply composer.json:2]: I added manually the suggest section in eclipse. After I have pushed it to git, the formatting was broken, so I formatted the composer.json correct.

**Comment by thexpand** [comment]
@DASPRiD Is there something more that has to be done here, so that it can be merged?

**Comment by nynka** [comment]
Hi guys! Excellent feature, any updates on this @DASPRiD?

File: composer.json (hunk 1 above)
Comment (DASPRiD) [reply composer.json:2]: Apparently your IDE added a lot of tabs in here. Please revert the formatting changes.

**Pull request closed without merging**